.github/workflows/main.yml

```diff
@@ -42,3 +42,36 @@ jobs:
           npm test
         env:
           DISPLAY: :99.0
+
+  build-on-macos-latest:
+    runs-on: macos-latest
```

Comment:
Is there a way we could use the build matrix support? It looks like the mac vs linux jobs are nearly identical. The main difference is how elixir is installed (does `actions/setup-elixir@v1` not support mac?), and installing xvfb

Reply:
> does actions/setup-elixir@v1 not support mac?

Yes. Right now, It does only support [linux](https://github.com/actions/setup-elixir/blob/05577888e77faf49acdd4166eb136960c209e8c0/src/setup-elixir.js#L45-L50).

> build matrix support

I want to support it too. I try to work on it later. But It might take a some time. So merging this PR at first and then gathering to one with using build matrix through other PR is also good way I think.

Reply:
Update: I've sent a PR to `actions/setup-elixir` to work on mac https://github.com/actions/setup-elixir/pull/17.

Reply:
FYI I closed that PR partially out of concern for the amount of time users will churn through running the Action on macOS—they'd be building Elixir/OTP from source on every workflow execution. For now, I've opened a [tracking issue](https://github.com/actions/setup-elixir/issues/23) to figure out what to do about macOS (and Windows, too, presumably).